backend/server/index.js

```diff
@@ -0,0 +1,28 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const app = express();
+const port = 3000;
+var cors = require('cors');
+
+let converter = require('../template-generation/generate-source-file.js');
+
+app.use(cors());
+app.use(bodyParser.json())
+app.use(
+    bodyParser.urlencoded({
+        extended: true,
+    })
+)
+
+/**
+ * Pass the template name in request.headers.template
+ * Example to run the test file: "key: test-template.js"
+ */
+app.post('/generate-file', (request, response) => {
+    converter.convertTemplate(request.headers.template, request.body);
```

Comment:
why pass the template in the header? it's probably easier in the body

Reply:
That's a good point, I've added the template to the JSON object as `data.template`